### PR TITLE
buffer: add end round and join miter

### PR DESCRIPTION
**Pull request by barendgehrels**

### CODE REVIEW

**Comment by barendgehrels** [comment]
Can this be reviewed? @vissarion @awulkiew ?

**Comment by vissarion** [comment]
> Can this be reviewed? @vissarion @awulkiew ?

Sorry for the delays, I had started but left it incomplete, will finish it today.

File: include/boost/geometry/strategies/geographic/buffer_join_round.hpp [hunk 1]
```diff
@@ -36,6 +34,20 @@ template
...
+    template <typename T>
+    using inverse = typename FormulaPolicy::template inverse
+        <
+            T, false, enable_azimuth, false, false, false
+        >;
+    template <typename T>
+    using direct = typename FormulaPolicy::template direct
+        <
+            T, enable_coordinates, false, false, false
+        >;
```
Comment (vissarion) [inline include/boost/geometry/strategies/geographic/buffer_join_round.hpp:49]: please see comment above

**Review by vissarion — APPROVED** [review]
Thanks! I am OK with merging. There is an issue about robustness when different inverse/direct formulas are used but that could be examined/addressed after merging.

**Pull request merged**